source/extensions/tracers/datadog/tracer.h

```diff
@@ -56,7 +56,7 @@ class Tracer : public Tracing::Driver, private Logger::Loggable<Logger::Id::trac
   explicit Tracer(const std::string& collector_cluster, const std::string& collector_reference_host,
                   const datadog::tracing::TracerConfig& config,
                   Upstream::ClusterManager& cluster_manager, Stats::Scope& scope,
-                  ThreadLocal::SlotAllocator& thread_local_slot_allocator);
+                  ThreadLocal::SlotAllocator& thread_local_slot_allocator, TimeSource& time_source);
```

Comment:
Please mention `time_source` in the contract above. I think it's reasonable to say that it's used to calculate request timeouts (could be used for something else in the future, but probably not).